clients/cobol-lsp-vscode-extension/src/services/snippetcompletion/cobolSnippets.json

```diff
@@ -96,55 +203,197 @@
+    "description": "Part of the IDENTIFICATION DIVISION. Note that the default option is EBCDIC"
+  },
+  "COMP": {
+    "prefix": "COMP",
+    "body": ["77 ${1:VAR-BIN} PIC S9(4) ${2|COMP,COMPUTATIONAL|}."],
+    "description": "BINARY data and synonyms COMP and COMP-4 are the two's complement data representation in COBOL. Two's complement Hex data (EBCDIC) representation of numeric values in fields within the DATA DIVISION."
+  },
+  "COMP-1": {
+    "prefix": "COMP-1",
+    "body": [
+      "77 ${1:VAR-FLOAT-SINGLE-PREC} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-1."
+    ],
+    "description": "A four-byte single-precision floating-point number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMP-2": {
+    "prefix": "COMP-2",
+    "body": [
+      "77 ${1:VAR-FLOAT-DOUBLE-PREC} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-2."
+    ],
+    "description": "A four-byte double-precision floating-point number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMP-3": {
+    "prefix": "COMP-3",
+    "body": [
+      "77 ${1:VAR-PACKED-NUM-DATA} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-3."
+    ],
+    "description": "Internally-packed (efficient) number declared in the DATA DIVISION. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=6-packed-decimal-comp-3"
+  },
+  "COMP-4": {
+    "prefix": "COMP-4",
+    "body": ["77 ${1:VAR-BIN} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-4."],
+    "description": "BINARY data and synonyms COMP and COMP-4 are the two's complement data representation in COBOL. Two's complement Hex data (EBCDIC) representation of numeric values in fields within the DATA DIVISION."
+  },
+  "COMP-5": {
+    "prefix": "COMP-5",
+    "body": ["77 ${1:VAR-BIN} PIC S9(4) ${2|COMP,COMPUTATIONAL|}-5."],
+    "description": "Typically used to communicate between Java/C/C++/Python/etc. and COBOL, with COMP-5, data, the size of the variable is defined by the field's PIC size. When numeric data is moved into a COMP-5 item, truncation occurs at the binary field size rather than at the COBOL picture clause declaration. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=arithmetic-formats-numeric-data"
+  },
+  "COMPUTE": {
+    "prefix": "COMPUTE",
+    "body": [
+      "COMPUTE ${1:NUM-VAR} OF ${2:DFHCOMMAREA} = ${3:WS-NUMVAR1} ${4|+,-,*,/,**|} ${5:WS-NUMVAR2}",
+      "\tON SIZE ERROR GO TO ${6:TRAP-OVERFLOW}",
+      "\tNOT ON SIZE ERROR GO TO ${7:PROGRAM-FINAL-ACTION}",
+      "END-COMPUTE",
+      "$0"
+    ],
+    "description": "COMPUTE provides an alebraic-style keyword for the speficiation of complex arithemetic expressions: + (add), - (subtract), * (multiply), / (divide), ** (power of) and any intrinsic functions that return numeric results. ON SIZE ERROR allows you to trap situcations such as divide by zero and result overflow. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=statements-compute-statement"
```

Comment:
"COMPUTE provides an algebraic-style keyword for the specification of complex arithmetic expressions: + (add), - (subtract), * (multiply), / (divide), ** (power of) and any intrinsic functions that return numeric results. ON SIZE ERROR allows you to trap situations such as divide by zero and result overflow. https://www.ibm.com/docs/en/cobol-zos/6.4?topic=statements-compute-statement"